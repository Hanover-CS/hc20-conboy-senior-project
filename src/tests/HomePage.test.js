import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import HomePage from '../HomePage'

describe('HomePage Tests', () => {
  test('home page renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<HomePage />, div)
    ReactDOM.unmountComponentAtNode(div)
  })


  test('snapshot of home page hasn\'t changed', () => {
    const component = renderer.create(<HomePage />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})