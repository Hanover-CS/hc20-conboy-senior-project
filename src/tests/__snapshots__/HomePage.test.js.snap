// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Tests snapshot of home page hasn't changed 1`] = `
<div
  className="HomePage"
>
  <h3>
    Welcome to AniTracker!
  </h3>
</div>
`;
